web-api/migration-terraform/main/lambdas/migration-segments.js

```diff
@@ -41,8 +44,8 @@ const migrateRecords = async ({
   items,
   ranMigrations = {},
 }) => {
-  if (!ranMigrations['bug-0035-private-practitioner-representing.js']) {
-    applicationContext.logger.info('about to run bug migration 0035');
+  if (!ranMigrations['bugMigration0035.js']) {
```

Comment:
I believe this should be testing for `bug-0035-private-practitioner-representing.js` — that’s what this line was before this PR, and I believe your intention was to revert the changes. 

Could you say more about why this should be `bugMigration0035.js` instead?